app/controllers/application_controller.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   include Pundit
-  protect_from_forgery with: :exception
+  protect_from_forgery with: :null_session
```

Comment:
The `Api::ApiController` doesn't subclass `ApplicationController` so this change to `protect_from_forgery` will not affect API requests. Consider changing the option back to `with: :exception` to carry on protecting the controllers that do subclass `ApplicationController`.